taxi/admin.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib import admin
+from .models import Manufacturer, Car, Driver
+
+# Register your models here.
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    pass
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ("model", )
+    list_filter = ("manufacturer", )
+
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
+    list_display = ("license_number", "username", "email", "first_name", "last_name", "password")
```

Comment:
The `password` field should not be included in `list_display` for security reasons. Consider removing it to prevent exposing sensitive information in the admin list view.